modules/images/webp-uploads/load.php

```diff
@@ -403,7 +403,29 @@ function webp_uploads_update_image_references( $content ) {
 	}
 
 	foreach ( $images as $img => $attachment_id ) {
-		$content = str_replace( $img, webp_uploads_img_tag_update_mime_type( $img, 'the_content', $attachment_id ), $content );
+
+		/**
+		 * Check for old uploaded images.
+		 */
+		if ( ! preg_match( '~src*=*["\'](.*?)["\']~', $img, $image_src ) ) {
+			continue;
+		}
+
+		if ( empty( $image_src ) ) {
+			continue;
+		}
+
+		/**
+		 * Get attachment id by the attachment url.
+		 */
+		$updated_attachment_id = attachment_url_to_postid( $image_src[1] );
```

Comment:
this can be an expensive query to run on every page load

Reply:
+1, I think unfortunately calling that function here, potentially numerous times on page load, is not feasible from a performance perspective.

Reply:
Yes. Thanks for the review.